core/api/service/author/error.cpp

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "api/extrinsic/error.hpp"
+#include "api/service/author/error.hpp"
 
 OUTCOME_CPP_DEFINE_CATEGORY(kagome::api, ExtrinsicApiError, e) {
```

Comment:
```suggestion
OUTCOME_CPP_DEFINE_CATEGORY(kagome::api, AuthorApiError, e) {
```
Maybe AuthorApiError?

Reply:
It will be removed